tmp-carp-client/client/src/index.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
```

Comment:
I don't think you meant to check this folder in

Reply:
Well, I did it in case someone wanted to checkout this branch and run it. Although currently there is no CI, it won't build without the dependency. Maybe it _would_ be better to put the code in a repo and point npm to it though, so I don't have to remember to untrack this later.